modules/appeals_api/app/workers/appeals_api/pdf_submit_job.rb

```diff
@@ -30,6 +30,8 @@ class PdfSubmitJob
     # 503 Database Offline || SOLR Service Offline || Intake API is undergoing maintenance
     RETRYABLE_EMMS_RESP_STATUS_CODES = [401, 429, 500, 503].freeze
 
+    STATSD_DUPLICATE_UUID_KEY = 'api.appeals.document_upload.duplicate_uuid'
```

Comment:
The `STATSD_UPLOAD_FAIL_KEY` for Appeals is set to `api.appeals.document_upload.fail`, so I was trying to keep things consistent here.